cdm/core/src/test/data/ncml/aggregationScanModifyAttributes.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<netcdf xmlns="http://www.unidata.ucar.edu/namespaces/netcdf/ncml-2.2">
+
+  <variable name="T">
+    <attribute name="new_attribute" type="string" value="new attribute value"/>
+    <attribute name="renamed_units" orgName="units"/> <!--TODO this causes an error when reading the ncml-->
```

Comment:
This TODO can now go
```suggestion
    <attribute name="renamed_units" orgName="units"/>
```

Reply:
Ahh yep, I missed that! Removed. Thanks :)